docs/cases/indicators-of-compromise.asciidoc

```diff
@@ -120,7 +120,13 @@ image::images/indicator-added-to-case.png[An indicator attached to a case]
 
 When you attach an indicator to a case, the indicator is added as a new comment with the following details:
 
-* *Indicator name*: Click the linked name to open the Indicator details flyout, which contains additional information about the indicator. Indicator details are in JSON format.
+* *Indicator name*: Click the linked name to open the Indicator details flyout, which contains the following informational tabs:
+** *Overview*: A summary of the indicator, including the indicator's name, the threat intelligence feed it came from, the indicator type, and additional relevant data.
```

Comment:
```suggestion
** *Overview*: A summary of the threat indicator, including its name and type, which threat intelligence feed it came from, and additional relevant data.
```
Just an idea, in case you like it